webpack/components/extensions/HostDetails/DetailsTabCards/HwPropertiesCard.js

```diff
@@ -41,8 +45,7 @@ const HwPropertiesCard = ({ isExpandedGlobal, hostDetails }) => {
   const cpuCount = facts?.['cpu::cpu(s)'];
   const cpuSockets = facts?.['cpu::cpu_socket(s)'];
   const coreSocket = facts?.['cpu::core(s)_per_socket'];
-  const reportedFacts = propsToCamelCase(hostDetails?.reported_data || {});
-  const totalDisks = reportedFacts?.disksTotal;
+  const blockDevices = facts?.blockdevices;
```

Comment:
If I look at Facter you have a fact `disks` which does exactly what you'd expect. Doesn't RHSM have something similar?

Reply:
Probably yes.